### ISSUE TITLE
Titles look wrong if multiline

### ISSUE THREAD

**Issue by arielj** [fastruby/points#116]
**Branch/Commit:**

Main branch.

**Expected behavior:**

The titles should not overlap when multiline and should not be too far apart either.

**Actual behavior:**

If the text is long enough to wrap to a new line, the line spacing looks bad. It's probably just a CSS line-height change.

**Steps to reproduce:**

Set a long project name (or a long project and subproject names), shrink the window width to force it to wrap to 2 lines.

**Screenshots and Videos**

![image](https://user-images.githubusercontent.com/188464/139137725-aadd4ab3-d8ae-43bd-9076-beeff33236d0.png)

**I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md)**

**Comment by taanguyen** [comment]
hi @lubc! thank you for offering to help! I have opened a PR. Please review it at your earliest convenience. Thank you!

**Comment by taanguyen** [comment]
Hi @lubc! I was able to fix the header, as detailed in the issue, but I haven't been able to fix another issue that has arisen: when the title is multiline, the header covers the content (as pointed out by @etagwerker). I'll move the conversation here since the PR thread is getting cluttered. I looked up a js way of adjusting the margin-top of an element. 
`function heightCalc(){
  topHeight = $('.header').height();
  $('dashboard-title-report').css({"margin-top": topHeight+"60px"});
} 

window.addEventListener('resize', heightCalc);
`
I tried adding this to project.js but that didn't help. Any advice on adding a js function?

**Comment by taanguyen** [comment]
I was able to link a separate js file to test project view. And pushing content down while resizing works but the title is sitting too low on the screen. I tried accounting for the padding of the fluid-container but that didn't work (or maybe I implemented it wrong). The custom directory inside javascripts is being included in //require custom/content in the `application.js`
![Screen Shot 2021-10-31 at 1 45 31 PM](https://user-images.githubusercontent.com/13709437/139600672-37651804-2901-4122-b27b-7e2bcaeec5c8.png)
![Screen Shot 2021-10-31 at 1 46 55 PM](https://user-images.githubusercontent.com/13709437/139600724-1d54173d-d383-4256-a081-587c7ce98583.png)

**Comment by taanguyen** [comment]
Hi @lubc! Sorry for all the emails. I followed your suggestions and the title doesn't get clobbered by the header anymore- no js necessary! thank you!! I forgot that position: fixed removes the element out of normal document flow. 

The only additional issue that came up was when scrolling, the title and the table get interspersed: 
![Screen Shot 2021-10-31 at 5 54 24 PM](https://user-images.githubusercontent.com/13709437/139607799-e3d3c877-b628-47c8-95d6-2449fff74706.png)

I know the CSS has position: sticky so I'm not sure if the intent was to keep the title on top at all times. To stop the title text and table text from jumbling, I made an extra change- I removed top: 53px. So when you scroll down, the two elements don't jumble 
![Screen Shot 2021-10-31 at 6 00 58 PM](https://user-images.githubusercontent.com/13709437/139608068-4914bd8f-26c8-453a-9e89-f3da2c7412ed.png). I hope removing the top offset was ok! Let me know if this was not a desirable modification! 
Here is the updated screenshot of working functionality (multiline title in the header doesn't cover the project title).
![Screen Shot 2021-10-31 at 6 03 27 PM](https://user-images.githubusercontent.com/13709437/139608142-78289f94-998b-4fe9-aa6f-91a9f27c3bab.png)
